game/CMakeLists.txt

```diff
@@ -81,6 +81,9 @@ find_package(LibEpoxy 1.2 REQUIRED)
 include_directories(${LibEpoxy_INCLUDE_DIRS})
 list(APPEND LIBS ${LibEpoxy_LIBRARIES})
 
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(deps REQUIRED IMPORTED_TARGET glib-2.0 gio-2.0 gobject-2.0)
+
```

Comment:
What for do we need pkgConf? This is mostlikely not installed within windows.

Reply:
Those find modules need pkgconfig, I believe.

Reply:
I haven't tried it again but when I originally made this PR, I had to get new find modules because not everything was being imported and these modules depende on PkgConfig.``